core/src/main/java/com/google/instrumentation/common/Timestamp.java

```diff
@@ -60,12 +64,14 @@ public static Timestamp create(long seconds, int nanos) {
 
   /**
    * Creates a new timestamp from given milliseconds.
+   *
+   * @return a new timestamp from
```

Comment:
Comment is truncated (from what?). Either delete the "from" or extend the sentence.